support-frontend/assets/helpers/abTests/abtest.ts

```diff
@@ -158,10 +169,10 @@ function userInBreakpoint(audience: Audience): boolean {
 
 	const minWidthMediaQuery = minWidth
 		? `(min-width:${breakpoints[minWidth]}px)`
-		: null;
+		: '';
 	const maxWidthMediaQuery = maxWidth
 		? `(max-width:${breakpoints[maxWidth]}px)`
```

Comment:
By changing the fallback value of the ternary operators for `minWidthMediaQuery` and `maxWidthMediaQuery` from `null` to an empty string it alters what this function will return if these fallbacks are used, as it'll potentially mean the `mediaQuery` variable is assigned empty string as a value instead of `null`. 

The console log below demonstrates how this will affect the return value of `window.matchMedia`:

<img width="287" alt="Screenshot 2021-11-15 at 16 35 38" src="https://user-images.githubusercontent.com/1590704/141819989-e77ae08d-5211-4c49-8e16-cbb02859e883.png">

I believe if we get to the point where we're calling `window.matchMedia` and `mediaQuery` is an invalid media query (such as an empty string) we should return `false`, you could put a conditional infront of the final return statement which would return `false` if `mediaQuery` is an empty string (or falsy)...

```
if (!mediaQuery) {
  return false;
}

return window.matchMedia(mediaQuery).matches;
```